nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/test/resources/templates/dummyNotificationToSendMail.ftl

```diff
@@ -0,0 +1,110 @@
+<html>
+<head>
+    <style>
```

Comment:
You don't need CSS for a dummy email.

Reply:
i used an exiting once and i cleaned it :), i will remove the css/table... parts